webapp-django/crashstats/signature/jinja2/signature/signature_reports.html

```diff
@@ -17,9 +17,13 @@
             {% for crash in query.hits %}
             <tr>
                 <td>
-                    <a href="{{ url('crashstats:report_index', crash.uuid) }}" class="external-link crash-id">
-                        {{ crash.uuid }}
-                    </a>
+                    <a href="{{ url('crashstats:report_index', crash.uuid) }}" class="external-link crash-id">{{ crash.uuid }}</a>
+                    {% if crash.cpu_info %}
+                        {% if crash.cpu_info.startswith('AuthenticAMD family 20 model 1') or
+                              crash.cpu_info.startswith('AuthenticAMD family 20 model 2') %}
```

Comment:
Duplication of code, but considering that signature report is going away, I guess it's fine.